.github/workflows/ci.yml

```diff
@@ -121,11 +121,17 @@ jobs:
           toolchain: nightly
           override: ${{ matrix.rust_release == 'latest-nightly' }}
 
-      - name: Run cargo test
+      - name: Run cargo test all targets (which does not include doctests)
+        uses: actions-rs/cargo@v1
+        with:
+          command: test
+          args: --no-fail-fast --all-targets
+
+      - name: Run cargo doc tests
         uses: actions-rs/cargo@v1
         with:
           command: test
-          args: --no-fail-fast
+          args: --no-fail-fast --doc
```

Comment:
do we need equivalents of this in wasm as well? does cargo doc run the doc tests?

Reply:
Maybe, but the tricky example tests have networking which doesn't generally work on wasm